explainability-core/src/main/java/org/kie/trustyai/explainability/model/tensor/Tensor1D.java

```diff
@@ -0,0 +1,46 @@
+package org.kie.trustyai.explainability.model.tensor;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Tensor1D<T> extends BaseTensor<T, T> {
+    public Tensor1D(List<T> data) {
+        dimensions = 1;
+        this.data = data;
```

Comment:
I would create a copy of the `data`: for example subList(...) returns a view of the original list so if I do `slice(...).fill(..) it can throw exception
```suggestion
        this.data = new ArrayList<>(data);
```